README.md

```diff
@@ -32,7 +31,14 @@ The service runs in the background consuming messages from Kafka. The messages a
 | HEALTHCHECK_INTERVAL         | 30s                               | Time between self-healthchecks (`time.Duration` format)
 | HEALTHCHECK_CRITICAL_TIMEOUT | 90s                               | Time to wait until an unhealthy dependent propagates its state to make this app unhealthy (`time.Duration` format)
 | IMAGE_API_URL                | http://localhost:24700            | The image api url
-| KAFKA_ADDR                   | "localhost:9092"                  | The address of Kafka (accepts list)
+| KAFKA_ADDR                   | `localhost:9092`                  | The address of (TLS-ready) Kafka brokers (comma-separated values)
```

Comment:
not sure I like the wording/change here, sorry:
the brokers are not "TLS-ready", they are either plaintext or TLS
it's the _app_ (this repo) which becomes _TLS-ready_ (i.e. ready to use TLS depending on its config) when you use the new dp-kafka

Reply:
Completely agree and done